flink-python/pyflink/table/table.py

```diff
@@ -621,6 +672,7 @@ def insert_into(self, table_path):
 
         :param table_path: The path of the registered :class:`TableSink` to which
```

Comment:
I prefer using full path in `:class:` tag, even `class:TableSink` is work well. What do you think?